builds/windows/desktop/10/windows.pkr.hcl

```diff
@@ -209,4 +209,22 @@ build {
       vsphere_folder           = var.vsphere_folder
     }
   }
+
+  dynamic "hcp_packer_registry" {
+    for_each = var.common_hcp_packer_registry_enabled ? [1] : []
+    content {
+      bucket_name = replace("${var.vm_guest_os_family}-${var.vm_guest_os_name}-${var.vm_guest_os_version}-${var.vm_guest_os_edition}", ".", "")
+      description = "${var.vm_guest_os_family} ${var.vm_guest_os_name} ${var.vm_guest_os_version} ${var.vm_guest_os_edition}"
```

Comment:
Should this be a local? e.g. `description = local.bucket_description`

Reply:
Done in commit f5d8ba04ffb9c2781c42aeb2ad231e419cf614b2